modules/react/text/stories/Text.stories.mdx

```diff
@@ -9,7 +9,7 @@ import {Title} from './examples/Title';
 
 import {TextComponent, LabelComponent, TypeLevelComponent} from './PropTables.splitprops.tsx';
 
-<Meta title="Preview/Text/React" />
+<Meta title="Components/Text/React" />
```

Comment:
I moved the Text stories out of Preview as per @RayRedGoose's [comment](https://github.com/Workday/canvas-kit/issues/1707#issuecomment-1244030794) here. They're now in a dedicated Text folder underneath Components:

![Screen Shot 2022-10-04 at 6 09 42 PM](https://user-images.githubusercontent.com/146020/193958046-bd08bd81-7af2-4cc5-a6ba-7041eff54e2e.png)

This would be consistent with where we plan on putting the Text components within the navigation hierarchy on the Canvas Site.

Note that this PR does not include the work for splitting `Text.stories.mdx` into separate MDX files for each Text component.

Reply:
The Text section will look different in the nav. See #1843.